core/certs.go

```diff
@@ -83,15 +88,30 @@ func getCertKeyPair(config *tomlConfig, secretBaseURL string, c *http.Client) (s
 		return "", "", errors.New(errStr)
 	}
 	defer resp.Body.Close()
-	collection := CertCollect{}
-	json.NewDecoder(resp.Body).Decode(&collection)
-	lc.Info(fmt.Sprintf("Successful on reading certificate from %s.", config.SecretService.CertPath))
-	lc.Info(fmt.Sprintf("\n %s \n \n %s", collection.Section.Cert, collection.Section.Key))
-	return collection.Section.Cert, collection.Section.Key, nil
+
+	collector := CertKeyCollector{}
+	json.NewDecoder(resp.Body).Decode(&collector)
+
+	switch resp.StatusCode {
+	case 200:
+		lc.Info(fmt.Sprintf("API Gateway TLS certificate/key found in Secret Store @/%s (%s)", config.SecretService.CertPath, resp.Status))
+		if debug {
+			lc.Info(fmt.Sprintf("\n %s \n \n %s", collector.Section.Cert, collector.Section.Key))
```

Comment:
I'd rather see this statement at "Debug" level, but since it's behind a debug flag from the command line, it's probably fine

Reply:
It's a very sensitive crypto asset, so having it triggered by a specific command line switch was more appropriate, as someone running "Debug" level just for other purposes would not see it by "accident".